cookbook/rule/general_information_on_rule_format.rst

```diff
@@ -0,0 +1,543 @@
+file and has to strictly follow the YAML format. You have to import a rule so it can be used in the PIM.
+
+This file starts with "rules" root element, which contains the list of enrichment rules. This document is about this
+list. Each rule is referred to by a code and can contain a list of conditions and actions.
+
+.. code-block:: yaml
+
+    # Example of a file with 2 rules
+    rules:
+        camera_set_canon_brand:
+            priority: 0
+            conditions:
+                - field: family.code
+                  operator: IN
+                  value:
+                    - camcorders
+                - field: name
+                  operator: CONTAINS
+                  value: Canon
+            actions:
+                - type: set_value
+                  field: camera_brand
+                  value: canon_brand
+        camera_copy_name_to_model:
+            priority: 0
+            conditions:
+                - field: family.code
+                  operator: IN
+                  value:
+                    - camcorders
+            actions:
+                - type: copy_value
+                  from_field: name
+                  to_field: camera_model_name
+
+Indentation is mandatory within the file and must be strictly identical to the one shown in the example.
+
+.. tip::
+
+    To apply a rule on every product, set the following condition:
+
+    .. code-block:: yaml
+
+        conditions:[]
+        actions:
+          - type:
+
+Enrichment Rule Structure
+-------------------------
+
+Structure’s elements which define a rule are :
+ - rule code (dynamic)
+ - priority*
+ - conditions
+ - actions
+
+Structure's elements which define a condition are :
+ - field
+ - locale​*
+ - scope​*
+ - operator
+ - value
+
+An enrichment rule is structured as follows :
+
+.. code-block:: yaml
+
+    [free rule code]:
+        priority​*:
+        conditions:
+            - field:
+              locale​*:
+              scope​*:
+              operator:
+              value:
+        actions:
+            - type:
+              [Diverse elements according to the action]
+
+Elements with * are optionals.
+
+**Dashes** - ​before element field and after each element contained in value part are mandatory.
+
+**Colon** : ​mandatory after each structure element.
+
+.. tip::
+
+    For more details you can see the `YAML specifications <http://yaml.org/spec/>`_.
+
+.. warning::
+
+    Rules code choice is up to you, however it has to contain only alphanumeric characters, underscores, dashes and be
+    less than 255 characters.
+
+A priority can be given to a rule. Priority will be considered for rules execution order. Without any given
+priority, rule has a zero-priority. The higher the priority, the sooner the rule will be executed.
+Therefore, a rule with 90 priority will be executed before rules with a 0 priority. If no rule has defined priority,
```

Comment:
as I understand, 90 is about the rank, it's not about having 90 priorities.. right ? in this case : "a rule with a 90-priority will be (...)"